yarn-project/simulator/src/avm/avm_simulator.test.ts

```diff
@@ -467,9 +468,10 @@ describe('AVM simulator: transpiled Noir contracts', () => {
         expect(results.output).toEqual([expectFound ? Fr.ONE : Fr.ZERO]);
 
         expect(trace.traceL1ToL2MessageCheck).toHaveBeenCalledTimes(1);
+        const foundOrUndefined = mockAtLeafIndex === undefined || results.output[0].equals(Fr.ONE);
```

Comment:
This is similar to what we do now for the `accrued_substate.test.ts`, because of the new way we handle non-exists criteria for leaf indices